policyengine_us/variables/gov/states/ga/tax/income/credits/ga_low_income_credit.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class ga_low_income_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Georgia low income credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.georgia.gov/document/document/2022-it-511-individual-income-tax-booklet/download"
+    defined_for = StateCode.GA
+
+    def formula(tax_unit, period, parameters):
+        # federal ajusted gross income
```

Comment:
Reorganize the code to follow the steps from the form and legal code:
1. Count exemptions
2. Add aged exemptions
3. Get amount per exemption
4. Return the product

Only get the parameters and AGI when you need them.